svo (install the zip, not me).xml

```diff
@@ -54203,7 +54203,16 @@ end
 
 -- returns a list of affs that tree and other basic cures can get
 function svo.gettreeableaffs()
-  return table.n_intersection(svo.keystolist(svo.affl), svo.treecurables)
+  local a = table.n_intersection(svo.keystolist(svo.affl), svo.treecurables)
```

Comment:
@keneanung svo.treecurableswithhypothermia wasn't public facing. Since we're changing the way that's handled though, I'm assuming you'd rather keep this the way it is.